dashboard/config/scripts/flappy.script

```diff
@@ -1,5 +1,5 @@
 id '6'
-title 'Flappy Code'
+title 'Flappy Bird'
```

Comment:
@Bjvanminnen wants this to show up in Google search results for "Flappy Bird".  We could special-case this script, but @laurelfan suggested just updating the name of the script to match.

<img src="https://cloud.githubusercontent.com/assets/413693/6478347/e8e5e8ec-c1e5-11e4-8bff-98fda5ae527e.png" width="500">